src/game/TacticalAI/AIMain.cc

```diff
@@ -69,7 +71,7 @@ void InitAI(void)
 #ifdef _DEBUG
 	if (gfDisplayCoverValues)
 	{
-		memset( gsCoverValue, 0x7F, sizeof( INT16 ) * WORLD_MAX );
+		std::fill_n(gsCoverValue, WORLD_MAX, 0x7F7F);
```

Comment:
What does `0x7F` actually stand for in this case?

Reply:
It's a magic debug number.